ferrilata-devilline.IntegrationTests/Scenarios/ApiControllers/Pitches/ApiPutPitchTests.cs

```diff
@@ -0,0 +1,152 @@
+                                    Encoding.UTF8,
+                                    "application/json");
+            request.Headers.Add("Authorization", "test");
+            var response = await _testContext.Client.SendAsync(request);
+
+            //Assert
+            Assert.Equal(System.Net.HttpStatusCode.OK, response.StatusCode);
+        }
+
+        [Theory]
+        [InlineData("api/pitch")]
+        public async Task PutPitchApi_IncorrectAuthentication_CorrectBody_ShouldReturnUnauthorized(string url)
+        {
+            //Arrange
+            var InputPitch = CreateNewPitch();
+            string InputJson = JsonConvert.SerializeObject(InputPitch);
+
+            //Act
+            var request = new HttpRequestMessage(HttpMethod.Put, url);
+            request.Content = new StringContent(InputJson,
+                                    Encoding.UTF8,
+                                    "application/json");
+            var response = await _testContext.Client.SendAsync(request);
+
+            //Assert
+            Assert.Equal(System.Net.HttpStatusCode.Unauthorized, response.StatusCode);
+        }
+
+        [Theory]
+        [InlineData("api/pitch")]
+        public async Task PutPitchApi_CorrectAuthentication_InCorrectBody_ShouldReturnNotFound(string url)
+        {
+            //Arrange
+            var InputPitch = new Pitch { BadgeName = "BadgeName", Status = "status", PitchMessage = "level" };
+            string InputJson = JsonConvert.SerializeObject(InputPitch);
+
+            //Act
+            var request = new HttpRequestMessage(HttpMethod.Put, url);
+            request.Content = new StringContent(InputJson,
+                                    Encoding.UTF8,
+                                    "application/json");
+            request.Headers.Add("Authorization", "test");
+            var response = await _testContext.Client.SendAsync(request);
+
+            //Assert
+            Assert.Equal(System.Net.HttpStatusCode.NotFound, response.StatusCode);
+        }
+
+        //DAOs
+
+        [Theory]
+        [InlineData("api/pitch")]
+        public async Task PutPitchApi_CorrectAuthentication_CorrectBody_ShouldReturnSuccess(string url)
+        {
+            //Arrange
+            var InputPitch = CreateNewPitch();
+            string InputJson = JsonConvert.SerializeObject(InputPitch);
+
+            //Act
+            var request = new HttpRequestMessage(HttpMethod.Put, url);
+            request.Content = new StringContent(InputJson,
+                                    Encoding.UTF8,
+                                    "application/json");
+            request.Headers.Add("Authorization", "test");
+            var response = await _testContext.Client.SendAsync(request);
+            var responseString = await response.Content.ReadAsStringAsync();
+
+            //Assert
+            Assert.Equal("Success", JsonConvert.DeserializeObject<Dictionary<string, string>>(responseString)["message"]);
+        }
```

Comment:
101 and all further Assert lines: no need to deserialize input string, enough to serialize the expected string:
Assert.Equal(JsonConvert.SerializeObject(new { message = "Unauthorized" }), ResponseBody);
This is Suvi's comment to similar code in my script